ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -96,14 +97,25 @@ def _parse_data_model(self):
                     schema = mnode["schema"] if mnode["schema"] else "default"
                     raw_sql = mnode.get("raw_sql", "")
                     description = mnode.get("description")
-                    dbt_user_name = cnode["metadata"].get("owner")
-                    user_name = f"*{dbt_user_name}*"
-                    user_fqn = fqn.build(
-                        self.metadata, entity_type=User, user_name=user_name
-                    )
-                    owner = self.metadata.get_entity_reference(
-                        entity=User, fqn=user_fqn
-                    )
+                    dbt_owner = cnode["metadata"].get("owner")
+                    owner = None
+                    if dbt_owner:
+                        owner_name = f"*{dbt_owner}*"
+                        user_owner_fqn = fqn.build(
+                            self.metadata, entity_type=User, user_name=owner_name
+                        )
+                        if user_owner_fqn:
```

Comment:
@OnkarVO7 how did this will go into else state?

Reply:
fqn will be created everytime, let's tackle this a little better where if we don't get any entity reference, we fallback to team

Reply:
@harshach so the logic is if we get the user then user will be ingested else the team will be ingested.

`fqn.build` method uses ES to search for the user. If the user is not found it will go into else

Reply:
why use es to look up user, lets use our apis
/api/v1/users/{name} should get you the user

Reply:
@harshach the reason behind using ES was to search the users a bit better. It might happen that the user name in the DBT catalog.json file might not be the exact same as the user name in OM